core/trino-main/src/main/java/io/trino/event/QueryMonitor.java

```diff
@@ -130,6 +132,7 @@ public QueryMonitor(
         this.serverVersion = requireNonNull(nodeVersion, "nodeVersion is null").toString();
         this.serverAddress = requireNonNull(nodeInfo, "nodeInfo is null").getExternalAddress();
         this.environment = requireNonNull(nodeInfo, "nodeInfo is null").getEnvironment();
+        this.instanceId = requireNonNull(nodeInfo, "nodeInfo is null").getInstanceId();
```

Comment:
Move `requireNonNull(nodeInfo, "nodeInfo is null")` to separate line and avoid repeating it